uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/services/leave-popup.service.js

```diff
@@ -47,9 +47,10 @@ define([
      * @param {String} leaveType
      * @param {String} selectedContactId - Contact ID for the contact dropdown
      *                                     when the manager/admin is opening the request
-     * @param {Boolean} forceRecalculateBalanceChange optional
+     * @param {Boolean} [forceRecalculateBalanceChange]
```

Comment:
Why is this inside square brackets?

Reply:
Optional param. http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values